app/src/pages/inside/dashboardPage/dashboardList/dashboardTable/hooks.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+import { fetch } from 'common/utils';
+
+export const useFetchedResponse = (url) => {
```

Comment:
If we store this hook in dashboards specific directory we assume this isn't a common thing.
And it also has an empty state as string that is not flexible enough.
So, for this case I suggest to form an url inside of the hook and name it more specific, e.g. as `useFetchDashboardConfig`.